packages/core/src/styles/Text.scss

```diff
@@ -55,6 +55,12 @@
         }
     }
 
+    &--pre {
+        .#{$prefix}-text__row {
+            white-space: pre;
```

Comment:
Should we use `pre` or `pre-wrap` in this case? If we use `pre`, text will only wrap on line breaks.

Reply:
right we should use `pre-wrap`. Nice catch, thanks